src/functions/accent.js

```diff
@@ -127,19 +126,19 @@ const htmlBuilder = (group, options) => {
             positionType: "firstBaseline",
             children: [
                 {type: "elem", elem: body},
-                {type: "elem", elem: accentBody},
+                {
+                    type: "elem",
+                    elem: accentBody,
+                    wrapperClasses: ["svg-align"],
+                    wrapperStyle: skew > 0
+                        ? {
+                            width: `calc(100% - ${2 * skew}em)`,
+                            marginLeft: `${(2 * skew)}em`,
+                        }
+                        : undefined,
+                },
             ],
         }, options);
-
-        // TODO: pass "svg-align" to stretchy.svgSpan
```

Comment:
This TODO seems to have gone away. Just want to make sure that was intentional. Similarly in accentunder.js